.gitignore

```diff
@@ -11,3 +11,6 @@ __pycache__
 
 # Linkchecker ignored files
 /.linkchecker
+
+# VSCode ignored files
+/.vscode
```

Comment:
You can add this to `.git/info/exclude` to ignore it in your local checkout without committing the ignore to the repo.

Reply:
Even better, just realized I can ignore `.vscode/settings.json` globally